From fb5cc9606195afc826d8dd2df28d9f66c2e5591b Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Sun, 3 Mar 2019 11:02:45 +0000
Subject: [PATCH] halimpl: Remove useless makefile

 * There's no point in having unused build targets when everyone uses
   the stock configs. Avoid the build warnings by removing the makefile.

Change-Id: I3a5d81ed1aa4cd0fc056d8fd83a1ee16e26244d6
---
 halimpl/Android.mk | 75 ----------------------------------------------
 1 file changed, 75 deletions(-)
 delete mode 100644 halimpl/Android.mk

diff --git a/halimpl/Android.mk b/halimpl/Android.mk
deleted file mode 100644
index 17b92d2..0000000
--- a/halimpl/Android.mk
+++ /dev/null
@@ -1,75 +0,0 @@
-# Copyright (C) 2011 The Android Open Source Project
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-ifeq ($(strip $(TARGET_USES_NQ_NFC)),true)
-
-LOCAL_PATH := $(call my-dir)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE       := libnfc-brcm.conf
-LOCAL_MODULE_TAGS  := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH  := $(TARGET_OUT_VENDOR_ETC)
-LOCAL_SRC_FILES    := $(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE       := libnfc-nci.conf
-LOCAL_MODULE_TAGS  := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH  := $(TARGET_OUT_VENDOR_ETC)
-LOCAL_SRC_FILES    := $(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE       := libnfc-nci_NCI2_0.conf
-LOCAL_MODULE_TAGS  := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH  := $(TARGET_OUT_VENDOR_ETC)
-LOCAL_SRC_FILES    := $(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE       := libnfc-brcm_NCI2_0.conf
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH  := $(TARGET_OUT_VENDOR_ETC)
-LOCAL_SRC_FILES    := $(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE       := libnfc-nxp.conf
-LOCAL_MODULE_TAGS  := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH  := $(TARGET_OUT_VENDOR_ETC)
-ifeq ($(strip $(NQ3XX_PRESENT)),true)
-LOCAL_SRC_FILES    := libnfc-nxp-PN80T_example.conf
-else
-LOCAL_SRC_FILES    := libnfc-nxp-PN66T_example.conf
-endif
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE       := libnfc-nxp_default.conf
-LOCAL_MODULE_TAGS  := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH  := $(TARGET_OUT_VENDOR_ETC)
-
-ifeq ($(strip $(NQ3XX_PRESENT)),true)
-LOCAL_SRC_FILES    := libnfc-nxp-PN80T_example.conf
-else
-LOCAL_SRC_FILES    := libnfc-nxp-PN66T_example.conf
-endif
-include $(BUILD_PREBUILT)
-
-endif
-- 
2.17.1

